playbook/app/pb_kits/playbook/pb_icon_circle/_icon_circle.tsx

```diff
@@ -26,23 +27,41 @@ type IconCircleProps = {
 }
 
 const IconCircle = (props: IconCircleProps) => {
-  const { aria = {}, className, dark = false, data = {}, icon, id, size = 'md', variant = 'default' } = props
+  const { aria = {}, className, dark = false, data = {}, emoji, icon, id, size = 'md', variant = 'default' } = props
 
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const classes = classnames(buildCss('pb_icon_circle_kit', size, variant), globalProps(props), className)
 
+  const isValidEmoji = (emoji: string) => {
+    // Using regular expression to check if the string is a valid emoji/emoji Unicode
+    const emojiRegex = /^(\p{Emoji}|\uFE0F)+$/u;
+    return emojiRegex.test(emoji);
+  };
+
   return (
     <div
         {...ariaProps}
         {...dataProps}
         className={classes}
         id={id}
     >
-      <Icon
-          dark={dark}
-          icon={icon}
-      />
+      {
+        icon && !emoji && (
```

Comment:
Not sure on this, but added in this check because without it, if a dev passes in icon AND emoji prop, both get rendered to the circle